deployments/UnityMixpanel/Assets/Mixpanel/Mixpanel.cs

```diff
@@ -552,10 +554,14 @@ private static mixpanel.detail.Mixpanel instance
         }
 
         static bool tracking_enabled = true;
+
         void Awake()
```

Comment:
Do we need to do everything on Awake()? I wonder if we can take advantage of Start() as well for things that don't rely on initialization or loading